README.md

```diff
@@ -187,6 +201,12 @@ Papers:
 
 ## Storage
 
+### Buffer Management
+
+- 2018, [Managing Non-Volatile Memory in Database Systems](https://db.in.tum.de/people/sites/vanrenen/papers/HyMem.pdf?lang=de), SIGMOD
+- 2018, [LeanStore: In-Memory Data Management Beyond Main Memory](https://db.in.tum.de/~leis/papers/leanstore.pdf), ICDE
+- 2020, [Umbra: A Disk-Based System with In-Memory Performance](https://db.in.tum.de/~radke/papers/hugejoins.pdf), CIDR
```

Comment:
Is the `Buffer Management` the main contribution of these papers?

Reply:
They are current state-of-art research, aiming to introduce a advanced buffer manager in Disk-based DBMSs to have the same level performance as In-Memory DBMSs.

For your question, I added the CMU lecture link and two old papers under this topic. 
The main contribution is **the 5 minite rule** by Jim Gray, which is included.

Reply:
got it. 👍